sunpy/coordinates/frames.py

```diff
@@ -469,7 +469,13 @@ class Helioprojective(SunPyBaseCoordinateFrame):
 
     rsun = QuantityAttribute(default=_RSUN, unit=u.km)
     observer = ObserverCoordinateAttribute(HeliographicStonyhurst)
+    
+    @property
+    def angular_radius(self):
```

Comment:
```suggestion
    def angular_radius(self):
        """
        Angular radius of the Sun as seen by the observer.

        The ``rsun`` frame attribute is the radius of the Sun in length units.
        The tangent vector from the observer to the edge of the Sun forms a
        right-angle triangle with the radius of the Sun as the far side and the
        Sun-observer distance as the hypotenuse.  Thus, the sine of the angular
        radius of the Sun is ratio of these two distances.
        """
```
Some wordage copied from the docstring of `sunpy.coordinates.sun.angular_radius()`